### PR TITLE
fixed codes on user profile

### PR DESCRIPTION
**Pull request by bittu1028**

### CODE REVIEW

File: src/__tests__/authService.test.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import {getItem} from '../storageService';
...
...IjoiaHR0cHM6Ly9pZGVudGl0eS1kZXYuc2FkZGxlYmFjay5jb20iLCJhdWQiOlsiaHR0cHM6Ly9pZGVudGl0eS1kZXYuc2FkZGxlYmFjay5jb20vcmVzb3VyY2VzIiwiY20tYXBpIiwiaW50cm9zcGVjdC1yZXNvdXJjZSJdLCJjbGllbnRfaWQiOiJoYy1tZW1iZXItZW5nYWdlbWVudCIsInN1YiI6Ijc3Nzk5MTAiLCJhdXRoX3RpbWUiOjE2MzU1MDU3ODEsImlkcCI6ImxvY2FsIiwidXJuOnNhZGRsZWJhY2s6dXNlcl9pZCI6WyI1MzcxZmUwZC0yMjQ4LTQ4OWYtYjgzNS04NGM5MmI4ODg5ZjIiLCI1MzcxZmUwZC0yMjQ4LTQ4OWYtYjgzNS04NGM5MmI4ODg5ZjIiXSwiaWQiOiI3Nzc5OTEwIiwiZW1haWxfdmVyaWZpZWQiOiJ0cnVlIiwidXJuOnNhZGRsZWJhY2s6cGVyc29uX2xpbmsiOiJVTktOT1dOIiwic2NvcGUiOlsib3BlbmlkIiwicHJvZmlsZSIsImVtYWlsIiwiaGM6Y29ubmVjdGlvbi1xdWVzdGlvbi1yZWFkIiwiaGM6Y29ubmVjdGlvbi1xdWVzdGlvbi1hbnN3ZXItcmVhZCIsImhjOmNvbm5lY3Rpb24tcXVlc3Rpb24tYW5zd2VyLWNyZWF0ZSIsImhjOmludGVybmFsLXN5c3RlbXMiLCJoYzpwZW9wbGUtY3JlZGVudGlhbHMtbWFuYWdlIiwiaGM6aW50cm9zcGVjdCIsIm9mZmxpbmVfYWNjZXNzIl0sImFtciI6WyJwd2QiXX0.YdjMBavxS2KQIKe_hvwX1GIGVaX0ePGcBDHL9KMKFsbYNkZQEHIlUii9LyPdxwcKshHhpjz4zBVYUk7_abhyu8TWd4gLRR_iG060jsz7qjY3s7oM18heLXpt-vwvAG6uGeBiLI_dCZQ0gGCcrRxuSEG_cNulxSY4hpj_D9a5btjwdgMHIgFDE5Y0bEvxNDa0KL9Nbu7awx4xG6w50j6frECYUsGLKs_9RQaXgLRsfbNP4ljiPYUqujey0yDIqfDqpT0XEm5mEMbE9HhIqX85GBk9SyiF54LRSuMJQsnNEXlTPnI4x1spYhY-9q00Ac_J3JUEwloROnrba5mT9n2RvA';
```
Comment (raoufr) [inline src/__tests__/authService.test.ts]: probably should not put this token in the code @bittu1028. Can we figure out a way to dynamically generate this maybe? let's discuss if you would like.

**Review by raoufr — CHANGES_REQUESTED** [review]

File: src/__tests__/authService.test.ts (hunk 1 above)
Comment (bittu1028) [reply src/__tests__/authService.test.ts]: @raoufr  we can put random string like “abcd” will that work for you ?

File: tests/__mocks__/react-native-app-auth.mock.js [hunk 2]
```diff
@@ -2,15 +2,12 @@ jest.mock('react-native-app-auth', () => ({
...
-                authorizeAdditionalParameters: {
-                    "session_state": "cfI5oHggpmOzlutQAgGl3ySdnHrsIsaeuf1xyfoZg1w.755408876312e573fdd85df0820c2d93"
-                }
+                idToken: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjEzODY4OTkxMzEsImlzcyI6ImppcmE6MTU0ODk1OTUiLCJxc2giOiI4MDYzZmY0Y2ExZTQxZGY3YmM5MGM4YWI2ZDBmNjIwN2Q0OTFjZjZkYWQ3YzY2ZWE3OTdiNDYxNGI3MTkyMmU5IiwiaWF0IjoxMzg2ODk4OTUxfQ.uKqU9dTB6gKwG6jQCuXYAiMNdfNRw98Hw_IWuA5MaMo",
```
Comment (raoufr) [inline tests/__mocks__/react-native-app-auth.mock.js:10]: @bittu1028 I am still concerned about having all of these tokens in here. Can we please connect and discuss on scrum before we merge? is this blocking you?

Comment (groberts314) [reply tests/__mocks__/react-native-app-auth.mock.js:10]: With a little bit of effort, you guys can probably construct some fake JWTs issued to a John Doe at johndoe@example.com, ID 1234567, etc.  Assuming you need these values to actually be JWTs and not just random strings for purposes of your test; you might not.  Dunno.  But just offering my two cents. ;-)

Comment (bittu1028) [reply tests/__mocks__/react-native-app-auth.mock.js:10]: @groberts314  i converted to fake tokens thanks @groberts314  for the feedback

**Review by raoufr — APPROVED** [review]
Looking good

**Pull request merged**